lib/mox.ex

```diff
@@ -318,6 +328,15 @@ defmodule Mox do
 
       Mox.defmock(MyMock, for: [MyBehaviour, MyOtherBehaviour])
 
+  ## Options
```

Comment:
Slightly unrelated, but I added this section so that the options are listed clearly and easy to see.